workshops/scrapbook_css/README.md

```diff
@@ -0,0 +1,249 @@
+
+  --colors-text: YOURCOLOUR; 
+
+  --colors-elevated: YOURCOLOUR;
+
+}
+```
+
+Replace `YOURCOLOUR` with the HEX codes you found, make sure to include the hash (hashtag) symbol at the start.
+
+You may still see some grey, these are elements that use the `--colors-muted` variable, can you work out how to change them based on what we learnt above?
+
+I choose these colours:
+
+<img src="https://cloud-loib1whnd.vercel.app/screenshot_2020-09-12_at_8.01.02_pm.png" width="380" alt="Green and purple themed Scrapbook">
+
+## Fancy fonts ❡
+
+What if your Scrapbook was in cursive? Or in a crazy techno font! Fonts can give your page a lot more character, so let’s switch them up.
+
+ is a great place to get started, scroll through all their fonts and find one you like! Click it and then for each of its styles click “Select this Style”. Then in the bar that has popped up on the right, click `Embed` and choose `@import`. Copy the first text box without the `<style>` tags. You should have something like:
+
+```
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;1,100;1,300&display=swap');
+```
+
+As we’ve most likely chosen different fonts, don’t worry if our links are different. Now paste this text snippet into the top of your code file (what you were writing on the right side).
+
+Similar to colours, Scrapbook uses CSS Variables to make it super simple for us to change the fonts. 
+
+Add the following code snippet just below the --colors-elevated line. 
+
+```css
+:root { 
+
+  --colors-background: YOURCOLOUR; 
+
+  --colors-text: YOURCOLOUR; 
+
+  --colors-elevated: YOURCOLOUR;
+  
+  /* Add this code below */
+  
+  --fonts-body: 'Font name', system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI';
+
+}
+```
+
+
+Replace ` ‘Font name’` with the name of your font, just keep the quotes! We’ve left the other fonts there as a backup for if the font we want won’t load.
+
+I went techno style, with the [Audiowide](https://fonts.google.com/specimen/Audiowide?category=Display&sidebar.open=true&selection.family=Audiowide) font.
+
+<img src="https://cloud-loib1whnd.vercel.app/screenshot_2020-09-12_at_8.01.02_pm.png" width="380" alt="Green and purple themed Scrapbook">
+
+## **Crazy CSS Animations - Part 1 ✌︎**
+
+We’ve got a fancy font and some magical colours but this site still needs some jazzing up! Looks animate it!
+First off, we’re going to make our text rainbow animated.
+
+Animations are made with keyframes, where you set the state of the object (in CSS properties) at different stages of the animation. If you've got any experience with video animation keyframing works very similarly here!
+
+Let’s define that animation:
+
+```css
+@keyframes rainbow {
+  14% {
+    color: #ff0000;
+  }
+  28% {
+    color: #ffa500;
+  }
+  42% {
+    color: #ffff00;
+  }
+  56% {
+    color: #008000;
+  }
+  70% {
+    color: #0000ff;
+  }
+  84% {
+    color: #4b0082;
+  }
+  100% {
+    color: #ee82ee;
+  }
+}
+```
```

Comment:
It'd be nice if English CSS color names were used here so it's easier for a beginner to understand what's happening with the keyframing.